boefjes/boefjes/plugins/kat_sec_txt/boefje.json

```diff
@@ -0,0 +1,14 @@
+{
+    "id": "security_txt",
+    "name": "Security.txt",
+    "description": "check",
+    "consumes": [
+        "Hostname"
```

Comment:
I'm not sure if consuming a Hostname is best behaviour, maybe `"HostnameHTTPURL", "IPAddressHTTPURL"` is more appropriate?

Currently I'm seeing findings such as `KAT-NO-SECURITY-TXT @ a.gtld-servers.net` and `KAT-NO-SECURITY-TXT @ alt1.aspmx.l.google.com`, even though no web server is running on those hostnames.

Reply:
Changed it to "HostnameHTTPURL"